bin/server.js

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env node
+
+var corsHeaders = require('hapi-cors-headers')
+var Hapi = require('hapi')
+var path = require('path')
+
+var plugin = require('../index')
+
+var server = new Hapi.Server()
+server.connection({ port: 3000 })
+
+server.register({
+  register: require('inert')
+}, function (error) {
+  if (error) {
+    throw error
+  }
+})
+
+// serve demo app at route path
+server.route({
+  method: 'GET',
+  path: '/{path*}',
+  handler: {
+    directory: {
+      path: path.resolve(__dirname, '../demo')
+    }
+  }
+})
+
+server.register({
+  register: require('good'),
+  options: {
+    reporters: [{
+      reporter: 'good-console',
+      events: { error: '*', log: '*', request: '*', response: '*', wreck: '*' }
+    }]
+  }
+}, function (err) {
```

Comment:
Little bit of inconsistency here as you seem to have used 'error' for everything else but use 'err' here :)